cmake/doxygen.cmake

```diff
@@ -7,6 +7,28 @@ find_package(Doxygen REQUIRED)
 
 # Configure doxyfile
 
+#####################################
+# Detect GCC include path to avoid message
+# /gstlearn/src/Space/SpacePoint.cpp:216: warning: include file iostream not found, perhaps you forgot to add its directory to INCLUDE_PATH?
```

Comment:
Is it because `#include <iostream>` @ SpacePoint.cpp:216 is inside the `namespace gstlrn` block?